MCDataProducts/inc/ProcessCode.hh

```diff
@@ -85,7 +85,7 @@ namespace mu2e {
 // stopped-muon physics processes, specific to Mu2e
       mu2eCeMinusEndpoint,    mu2eCeMinusLeadingLog,   mu2eCePlusEndpoint,  mu2eDIOLeadingLog, // 170
       mu2eInternalRMC,        mu2eExternalRMC,         mu2eFlateMinus,      mu2eFlatePlus, // 174
-      mu2eFlatPhoton, // 175
+      mu2eFlatPhoton, Mu2eSpecialCutsProcess, // 176
```

Comment:
I prefer to group the process codes by function, so either find a defunct value earlier in the 'G4' area and reuse it, or put in a gap (Mu2eSpecialCutsProcess=200)

Reply:
Would it not be inconsistent with the current policy stated in that file?
It seems it would also require putting some fillers in the PROCESSCODE_NAMES macro, to make sure name() works well.

Reply:
Please rename it mu2eSpecialCutsProcess (to be consistent with the other 'mu2e' codes), and replace
"unused001" with that.  That keeps these 'special' codes together.

Reply:
About Krzysztof's comments on the policy stated in the file.   Since we are making a clean break with backwards compatibility, reuse of an enum value is OK.  We never wrote that down but we did talk about it.

Having said that, there is no real organization to these codes.   We never tried to make a design that would allow for it - every time that we commission a new G4 we have to add a few new codes to the end and every time that we add a new mu2e process, we have to add it to the end.    So, even if we reorganized them now to put all of the mu2e codes together, it will break again soon.

When I wrote ProcessCode we only had support for enum matched to string classes with dense enums.  But now we have support for sparse enums.   We could consider switching this to a sparse enum and reorganizing the enum values?  We could define Mu2e codes starting at 1 and G4 code starting at 1000, say.  The reason I hesitate on this is how do we audit the code to make sure that no one has defined a test on range of codes?  I explicitly did not include operators < and > but enum types are implicitly convertible to ints.  Also someone might have done this in an tuple analysis where the type safety and the names are all gone.